scripts/clean-local-binary.sh

```diff
@@ -2,9 +2,9 @@
 
 # no `set -e` here as we allow commands to fail in this script
 
-TMPDIR=${TMPDIR:-"/tmp/parachain_dev*"}
+LITENTRY_PARACHAIN_DIR=${LITENTRY_PARACHAIN_DIR:?}
```

Comment:
Shouldn't we have the default/fallback path if we don't set it? This is particularly useful when we call it via the `make` like `make clean-docker-rococo`.

Reply:
No, that's yet another way how errors end up getting masked because things work by accident until they don't :P
In t-shirt form, "defaults are bug eggs"